### DISCUSSION TITLE
Resolve tsconfig

### DISCUSSION THREAD

**Discussion by black7375** [vitejs/vite-plugin-react-swc#92] [Ideas]
The following project seems to be resolve.
- [unplugin-swc](https://github.com/egoist/unplugin-swc) (Use [load-tsconfig](https://github.com/egoist/load-tsconfig))
- [tsconfck](https://github.com/dominikg/tsconfck)

Do you plan to resolve tsconfig in this project?

**Comment by ArnaudBarre** [comment]
For now I don't plan to do so. There are few options that are important and I want to encourage people to use spec compliant transformation. The only one that can have an impact and could be infered is jsx import source, but I think the the duplication of this config is beneficial for the simplicity and the speed of the plugin

**Reply by black7375** [reply]
I got it. thanks to quick reply!!
